tests/unit.rs

```diff
@@ -86,6 +86,29 @@ fn new_path_windows_fun() {
     }
 }
 
+#[test]
+fn to_file_path_for_relative_windows_paths() {
+    if cfg!(windows) {
+        let url = Url::parse("file://example.com/C:foo").unwrap();
+        assert!(url.to_file_path().is_ok());
```

Comment:
NB: unlike other tests, this is `ok`. I would think that it should be `err` as well, but `std::Path` disagrees with me and insists that `\\exaple.com\C:foo` is an absolute path. Is this a bug in stdlib maybe as well?

Reply:
@retep998 perhaps you know? Consider this program:

```rust
use std::path::PathBuf;

fn main() {
    let p1 = PathBuf::from(r"C:foo");
    let p2 = PathBuf::from(r"\\example.com\C:foo");
    println!("{} {}", p1.is_absolute(), p2.is_absolute());
}
```

It currently prints `false true`. Is this the correct behavior?

Reply:
`C:foo` is *relative* to the current directory for the `C` drive. `\\example.com\C:foo` is a UNC path which is absolute.

For more information on the types of paths... https://googleprojectzero.blogspot.com/2016/02/the-definitive-guide-on-win32-to-nt.html